extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -0,0 +1,114 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import 'mocha';
+import * as azdata from 'azdata';
+import * as utils from './utils';
+const path = require('path');
+import { context } from './testContext';
+import assert = require('assert');
+import { getStandaloneServer } from './testConfig';
+import { exists, readFileSync, unlinkSync } from 'fs';
+
+if (context.RunTest) {
+	suite('Schema compare integration test suite', () => {
+		suiteSetup(async function () {
+			await utils.sleep(5000); // To ensure the providers are registered.
+			console.log(`Start schema compare tests`);
+
+		});
+		test('Schema compare dacpac comparision and script generation', async function () {
+			let service = await azdata.dataprotocol.getProvider<azdata.SchemaCompareServicesProvider>('MSSQL', azdata.DataProviderType.SchemaCompareServicesProvider);
+			assert(service, 'Schema Compare Service Provider is not available');
```

Comment:
I see this repeated in almost every test, maybe make it a method?

Reply:
moved at test suite setup to create this.